site/docs/v1/tech/identity-providers/samlv2/azure-ad.adoc

```diff
@@ -0,0 +1,179 @@
+
+image:identity-providers/azure-ad-saml/email-claim[Setup custom email claim,width=1200,role=shadowed]
+
+== Finish Setup of Azure AD Application
+
+With the Identity Provider setup complete, the next step is to input our newly created values into our AzureAD Application.
+
+Navigate to your AzureAD application. Under step one, click on the pencil to edit.
+
+image:identity-providers/azure-ad-saml/saml-5-steps-overview[SAML 5 steps overview,width=1200,role=shadowed]
+
+On this screen add the Entity ID and Reply URL. The Entity ID can be obtained by going to `Setting > Identity Providers > your SAML Idp > Magnifying Glass Icon > SAML v2 Integration details`. This value is synonymous with the Issuer value in FusionAuth.
+
+image:identity-providers/azure-ad-saml/integration-details[Integration Details,width=1200,role=shadowed]
+
+== Download and Import X509 Certificate
+
+Previously, we created and used a "dummy" X509 certificate. At this point, we can change this. From the overview of your SAML application in AzureAD, under step three, you should find a Certificate (Base64) to download.
+
+image:identity-providers/azure-ad-saml/generated-certificate[Generated Certificate,width=1200,role=shadowed]
+
+Next, import this certificate into Key Master (`Settings > Key Master > Import Certificate`). Leave the Key identifier property blank, as this will be autogenerated from the thumbprint in the existing certificate.
+
+Update the certificate used on your Idp configuration (`Settings > Identity Providers > your SAML IDP > Verification Key`) with this new X509 certificate. You can also remove the "dummy" certificate from Key Master once complete.
+
+[WARNING]
+====
+Leave the Key identifier property blank, as this will be autogenerated from the thumbprint of the existing certificate. Otherwise, FusionAuth will not be able to validate the signature of the AuthN response from AzureAD when using this integration.
+====
+
+== Assign Users
+
+[NOTE]
+====
+If you are using an existing AzureAD application with assigned users, you may skip this step.
+====
+
+Before we can complete our integration, assign AzureAD users to your application. Add a user by navigating to `Users and groups` and clicking on the `Add user/group` button. Follow onscreen instructions to select and add a user.
+
+image:identity-providers/azure-ad-saml/add-user-azure[Add a User to Azure,width=1200,role=shadowed]
+
+== Complete the Login
+
+image:identity-providers/azure-ad-saml/new-login-button[New Login With Azure-Button,width=1200,role=shadowed]
+
+At this point, what remains is to attempt a login. To test quickly, navigate to `Applications > Your Application > View (magnify icon) > OAuth2 & OpenID Connect Integration details > Login URLs`. Copy is this URL.
+
+Open a new tab, paste in the login URL, and you should see your new SAML AzureAD IdP login option. If your integration fails, remember to review the Event Logs (`System > Event Log`)
```

Comment:
```suggestion
Open a new tab, paste in the login URL, and you should see your new SAML AzureAD IdP login button. 

Click on that button, log in with one of the Azure AD users you configured above, and you should arrive back at your redirect URL.

If your integration fails, remember to review the Event Logs (`System > Event Log`)
```

Did you show them how to configure the redirect URL in the FusionAuth application?